diff --git a/dist/module.mjs b/dist/module.mjs
index 8f80a9b9dee5fdfa22525854e955e395416432a0..0815d09496d42ec358c8e5877f376d3e86dd0871 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -289,7 +289,7 @@ const apiToken = ${JSON.stringify(connection.apiToken)}
 async function d1HttpDriver(sql, params, method) {
   if (method === 'values') method = 'all'
 
-  const { errors, success, result } = await $fetch(\`https://api.cloudflare.com/client/v4/accounts/\${accountId}/d1/db/\${databaseId}/raw\`, {
+  const { errors, success, result } = await $fetch(\`https://api.cloudflare.com/client/v4/accounts/\${accountId}/d1/database/\${databaseId}/raw\`, {
     method: 'POST',
     headers: {
       Authorization: \`Bearer \${apiToken}\`,
