diff --git a/dist/module.mjs b/dist/module.mjs
index cb0df587e07404f133284b5f8d94c2a9ac46e000..751da1b826bcf7bdea7077fca12f2151c9d6ac94 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -446,7 +446,7 @@ const apiToken = ${JSON.stringify(connection.apiToken)}
 async function d1HttpDriver(sql, params, method) {
   if (method === 'values') method = 'all'
 
-  const { errors, success, result } = await $fetch(\`https://api.cloudflare.com/client/v4/accounts/\${accountId}/d1/db/\${databaseId}/raw\`, {
+  const { errors, success, result } = await $fetch(\`https://api.cloudflare.com/client/v4/accounts/\${accountId}/d1/database/\${databaseId}/raw\`, {
     method: 'POST',
     headers: {
       Authorization: \`Bearer \${apiToken}\`,
