diff --git a/dist/runtime-core.cjs.js b/dist/runtime-core.cjs.js
index 71a044f550e1cd2fb65bc87a64eb39d155f02827..074482f4f3153ba412e78c0e79be2efb7a44cb38 100644
--- a/dist/runtime-core.cjs.js
+++ b/dist/runtime-core.cjs.js
@@ -1444,7 +1444,7 @@ function patchSuspense(n1, n2, container, anchor, parentComponent, namespace, sl
     }
   }
 }
-let hasWarned = false;
+let hasWarned = true;
 function createSuspenseBoundary(vnode, parentSuspense, parentComponent, container, hiddenContainer, anchor, namespace, slotScopeIds, optimized, rendererInternals, isHydrating = false) {
   if (!hasWarned) {
     hasWarned = true;
