diff --git a/dist/rollup/index.mjs b/dist/rollup/index.mjs
index 9aadcbc7925e965062670f1d86436f8e0830b200..996e3de96b770476973e2b897caabed832ea4b24 100644
--- a/dist/rollup/index.mjs
+++ b/dist/rollup/index.mjs
@@ -1185,7 +1185,7 @@ function astToObject(node) {
       return obj;
     }
     case "ArrayExpression": {
-      return node.elements.map((el) => astToObject(el)).filter(Boolean);
+      return node.elements.map((el) => astToObject(el)).filter((el) => el !== null && el !== void 0);
     }
     case "Literal": {
       return node.value;
