diff --git a/dist/rollup/index.mjs b/dist/rollup/index.mjs
index 5f2bc76d97385f8f0af55ff0643f57f5b3d05c34..f9fd3eb5a908b62907a0509a3c6b00fb700072b4 100644
--- a/dist/rollup/index.mjs
+++ b/dist/rollup/index.mjs
@@ -1185,7 +1185,7 @@ function astToObject(node) {
       return obj;
     }
     case "ArrayExpression": {
-      return node.elements.map((el) => astToObject(el)).filter(Boolean);
+      return node.elements.map((el) => astToObject(el)).filter((el) => el !== null && el !== void 0);
     }
     case "Literal": {
       return node.value;
